README.md

```diff
@@ -307,6 +308,21 @@ yum -q makecache -y --disablerepo='*' --enablerepo='lightbits-discovery-client'
 yum install discovery-client
 ```
 
+## PR Checks
+
+### How-to
+
+1. **Non-Organization Users:**
+   - Fork the repository
+   - Make the change you want
+   - Open a pull request with the public repo (not the fork) as the base
+   - A person inside the organization will review the pull request and, if it looks fine, he'll trigger the PR check on the pull request branch
```

Comment:
Change "a person inside the org" to "a committer from Lightbits"
Change "he" to "they"